scripts/fix-thrift-globals.js

```diff
@@ -0,0 +1,37 @@
+const fs = require("fs")
```

Comment:
This script inserts `var ` and `window.` for node and browser thrift files respectively. Otherwise Node at least gets mad.